adb/fastboot.py

```diff
@@ -99,7 +100,7 @@ def HandleSimpleResponses(
           info_cb: Optional callback for text sent from the bootloader.
 
         Returns:
-          OKAY packet's message.
+          Tuple - OKAY packet's message, List of preceding Fastboot Messages
         """
```

Comment:
Instead of Tuple - just write it as a tuple ('OKAY', packet's message', list of preceding fastboot messages)

Reply:
Everywhere else too

Reply:
Done